ci/shared.sc

```diff
@@ -4,7 +4,7 @@
   * via import $file
   */
 
-import $ivy.`org.scalaj::scalaj-http:2.4.1`
+import $ivy.`org.scalaj::scalaj-http:2.4.2`
```

Comment:
Just a thought. Would it be possible to move the `Deps` object into its own file, include it here (and in `build.sc`) and then use `Deps.scalaj` (or in essence it's value) in this ivy-import?

Reply:
Not sure, that would require using Ammonite's multi-stage scripts which I don't remember if Mill supports or not